### DISCUSSION TITLE
How to implemente gift car

### DISCUSSION THREAD

**Discussion by YimMenuTheBest** [YimMenu/YimMenu#1153] [Q&A]
I would like to know how I could implement gifting cars, I know it is using DECORATOR::DECOR_SET_INT(veh, "Player_Vehicle", NETWORK::_NETWORK_HASH_FROM_PLAYER_HANDLE(player));, 

could someone help me to implement?
  
Thanks in advance to this beautiful community

**Comment by Tayipc38** [comment]
You need some more decorators and a small trick involving "Replacement" with full garages. You can see what decorations exist within the game scripts and their string values are rather explanatory. With that said, this does classify as recovery since the car can be sold afterwards (if that isnt patched yet).
